packages/components/src/components/ScalarMenu/ScalarMenuLink.vue

```diff
@@ -0,0 +1,15 @@
+<script setup lang="ts">
+import type { Component } from 'vue'
+
+defineProps<{
+  is?: string | Component
+}>()
+</script>
+<template>
+  <component
```

Comment:
These dynamic components don't work so grate on SSR. I think the issue was with `as`

Reply:
Oh hmmmm, we use to we can render a `RouterLink` sometimes 🤔 I should happen at "compile time" though

Reply:
This might be fine, cant really test nuxt rn anyway. yolo

Reply:
It should be fine. We use dynamic components elsewhere.

Reply:
Yea and a few times it didn't work. I had to change it. It's a specific way though